copy edit of docs (spelling, grammar, and clarity) (#44)
Mirantis/launchpad: docs/configuration-file.md

# Mirantis Launchpad Configuration File

> Mirantis Launchpad cluster configuration is described in YAML format. You can
> create and modify yaml files using your favorite text editor. The default name
> for this file is launchpad.yaml, although other file names could be used.

Reviewer: ```suggestion
Mirantis Launchpad cluster configuration is described in YAML format. You can create and modify yaml files using your favorite text editor. The default name for this file is launchpad.yaml, although other file names could be used.
```

Fixed linefeeds.

    installURLWindows: https://get.mirantis.com/install.ps1
```

We follow Kubernetes-like versioning and grouping in launchpad configuration so you'll see familiar attributes such as `kind`.

## `apiVersion`

> Launchpad currently supports `launchpad.mirantis.com/v1beta1`, `v1beta2`, and
> `v1beta3`. Earlier configuration file syntaxes should still work but
> will not have support for changes and additions in new versions.

Reviewer: ```suggestion
The latest API version is `launchpad.mirantis.com/v1beta3`, but earlier configuration file syntaxes should still work without support for the changes and additions in later versions.
```

    installURLWindows: https://get.mirantis.com/install.ps1
```

> We follow Kubernetes-like versioning and grouping in launchpad configuration
> so you'll see familiar attributes such as `kind`.

Reviewer: ```suggestion
We follow Kubernetes-like versioning and grouping in launchpad configuration so you'll see familiar attributes such as `kind`.
```
